components/feature/sitepermissions/src/main/java/mozilla/components/feature/sitepermissions/SitePermissionsFeature.kt

```diff
@@ -161,7 +159,7 @@ class SitePermissionsFeature(
         permissionRequest: PermissionRequest,
         optionalSessionId: String? = null
     ) {
-        val thisSessionId = optionalSessionId ?: sessionId
+        val thisSessionId = optionalSessionId ?: getCurrentTabState()?.id
```

Comment:
Here we need to warranty that when the provided `sessionId` is not found we have to fallback to selected tab.